opnode/rollup/derive/payload_attributes.go

```diff
@@ -328,14 +334,3 @@ func DeriveDeposits(l1Info L1Info, receipts []*types.Receipt) ([]l2.Data, error)
 	}
 	return encodedTxs, nil
 }
-
-type BatchData struct {
-	Epoch     rollup.Epoch // aka l1 num
-	Timestamp uint64
-	// no feeRecipient address input, all fees go to a L2 contract
-	Transactions []l2.Data
-}
-
-func ParseBatches(data l2.Data) []BatchData {
```

Comment:
Parsing multiple batches from the same transaction call-data would be great to have. There are many L2 blocks (one batch each), we should be bundling them.